pkg/addons/vpc_controller.go

```diff
@@ -251,6 +251,10 @@ func (v *VPCController) deployVPCResourceController() error {
 		// If an OIDC provider isn't associated with the cluster, the VPC controller relies on the managed policy
 		// attached to the node role for the AWS VPC CNI plugin.
 		sa := &corev1.ServiceAccount{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "ServiceAccount",
+				APIVersion: "v1",
+			},
 			ObjectMeta: metav1.ObjectMeta{
```

Comment:
could we use the `kubernetes.NewServiceAccount` func we have instead? Means any future changes to `APIVersion` is scoped to one place 😄

Reply:
Good call. I was looking for something in the standard K8s library, didn't know we had a helper for this.